app/controllers/events_controller.rb

```diff
@@ -65,6 +65,11 @@ def destroy
     redirect_to events_path
   end
 
+  def event_attendance
```

Comment:
if we're being RESTful we might prefer not to create new actions in the controller and instead prefer to create a new attendance object, but you've got the RESTful route set up, so this seems a good intermediate if we can set up the refactoring/feature ticket that would pull attendance out into a different model ...